crates/scarb-api/src/lib.rs

```diff
@@ -75,64 +78,121 @@ fn artifacts_for_package(path: &Utf8Path) -> Result<StarknetArtifacts> {
     Ok(starknet_artifacts)
 }
 
+#[derive(PartialEq, Debug)]
+struct ContractArtifactData {
+    path: Utf8PathBuf,
+    test_type: Option<String>,
+}
```

Comment:
This name is bad imo, there are already a lot of structs in this file that are named similarly, but have different semantics. Maybe there is another way to express in the typesystem that one of the paths should be treated as "default/base" (should be documented what it actually means here) one?

```rs
struct StarknetArtifactsFiles {
    default_artifacts_file: Utf8PathBuf,
    other_files: Vec<Utf8PathBuf>,
}
```

then it could have `load_contracts_artifacts` as a method. WDYT?

Reply:
Refactored